components/TextControls/src/BaseTextAreas/MaterialTextControls+BaseTextAreas.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
This umbrella header is a bit odd. Are the styles that we expect clients to use going to have extensions style +'s in them? Is this because we wanted to group our textControls together?

Reply:
Our decision to split text controls up into a bunch of little targets has resulted in some of the umbrella headers for these targets being pretty small. And yes, the targets have been structured as extensions. This was signed off on in https://github.com/material-components/material-components-ios/pull/9430.

Reply:
ok. Lets us make sure this is clear in our copyable code snippets docs.

Reply:
The migration guide for text fields (which will eventually have to be updated to account for text areas) has copyable snippets for including TextControls in a BUILD file and for importing them